lib/workers/repository/dependency-dashboard.ts

```diff
@@ -254,6 +257,13 @@ export async function ensureDependencyDashboard(
     }
     issueBody += '\n';
   }
+
+  const warn = getDepWarningsDashboard(packageFiles);
+  if (warn.length) {
+    issueBody += warn;
```

Comment:
check max issue body length/size ?

Reply:
after preparing the dependency dashboard body function `smartTruncate()` checks the body length and changes it if needed.

Flow:
1.
https://github.com/renovatebot/renovate/blob/6246a61bf4a4050e07632fe2658f774528c1e121/lib/workers/repository/dependency-dashboard.ts#L359-L367
2.
https://github.com/renovatebot/renovate/blob/6246a61bf4a4050e07632fe2658f774528c1e121/lib/modules/platform/github/index.ts#L1575
3.
https://github.com/renovatebot/renovate/blob/6246a61bf4a4050e07632fe2658f774528c1e121/lib/modules/platform/utils/pr-body.ts#L8-L33